src/io/files.js

```diff
@@ -1125,6 +1125,7 @@ p5.PrintWriter = function(filename, extension) {
     this.content += data + '\n';
   };
   /**
+   * Flushes the PrintWriter object that calls it
```

Comment:
I would actually go for something more descriptive, as "flush" means nothing to someone unfamiliar with a PrintWriter. Something like "Clears the data already written to the PrintWriter"